modules/TestUtils.js

```diff
@@ -88,7 +88,7 @@ export const objectContains = (object, value, compareValues) => {
     compareValues = isEqual
 
   return Object.keys(value).every(k => {
-    if (isObject(object[k])) {
+    if (isObject(object[k]) && Object.keys(value[k]).length > 0) {
```

Comment:
i wonder if it would be cleaner to move this check up to line 89? That way it would be handled both in the recursion, and the first time this function is called at the top of the stack. (If we did, we'd have to handle ensuring both `value` and `object` have keys to compare)